keps/sig-node/2371-cri-pod-container-stats/README.md

```diff
@@ -808,7 +808,8 @@ We expect no non-infra related flakes in the last month as a GA graduation crite
 - Conformance tests for the fields in `/metrics/cadvisor` should be created.
 - Validate performance impact of this feature is within allowable margin (or non-existent, ideally).
 	- The CRI stats implementation should perform better than they did with CRI+cAdvisor.
-- cAdvisor stats provider may be marked as deprecated (depending on stability of new CRI based implementations).
+- cAdvisor stats provider will be marked as deprecated, as well as the cAdvisor providing the metrics endpoint `/metrics/cadvisor`.
```

Comment:
can we add a goal of providing extensive migration docs for people to migrate to the new metrics from cxAdvisor?

Reply:
yes good idea!

Reply:
done

Reply:
Sorry for coming back to this. I was thinking about it and I think it will be premature to mark those endpoints as deprecated if the only alternative is an API in beta status. @deads2k as PRR approver for the opinion.

Reply:
to me "deprecated" means "in the process of being removed, but still fully functional". If we waited until GA to begin the deprecation process, then the deprecation process would outlive the associated feature gate and KEP. Beginning in beta is a good way to ease into the process.

Reply:
Let's keep it as deprecated now and maybe fix it after SIG Arch discussion next Thursday. @dchen1107 @derekwaynecarr in case you have concern with this approach